src/Raven.Server/Smuggler/Documents/StreamSource.cs

```diff
@@ -640,6 +639,21 @@ public async Task<DatabaseRecord> GetDatabaseRecordAsync()
                     }
                 }
 
+
+                if (reader.TryGet(nameof(databaseRecord.Sharding), out BlittableJsonReaderObject sharding) &&
+                    sharding != null)
+                {
+                    try
+                    {
+                        databaseRecord.Sharding = JsonDeserializationCluster.ShardingConfiguration(sharding);
+                    }
+                    catch (Exception e)
```

Comment:
I wonder if we should fail here entirely
cc @ayende

Reply:
I think so?  what will happen if we continue in this case?

Reply:
done